api/app/wildfire_one.py

```diff
@@ -117,15 +117,44 @@ def use_wfwx():
     return using_wfwx
 
 
+def _create_redis():
+    return StrictRedis(host=config.get('REDIS_HOST'),
+                       port=config.get('REDIS_PORT', 6379),
+                       db=0,
+                       password=config.get('REDIS_PASSWORD'))
+
+
 async def _fetch_access_token(session: ClientSession) -> dict:
     """ Fetch an access token for WFWX Fireweather API
     """
     logger.debug('fetching access token...')
     password = config.get('WFWX_SECRET')
     user = config.get('WFWX_USER')
     auth_url = config.get('WFWX_AUTH_URL')
-    async with session.get(auth_url, auth=BasicAuth(login=user, password=password)) as response:
-        return await response.json()
+    cache = _create_redis()
+    key = f'{auth_url}?{urlencode({user: user, password: password})}'
```

Comment:
Should we be storing these in plaintext in Redis?

Reply:
Keying by `auth_url` should be enough, no?

Reply:
Re. plaintext - we're using secure redis, so I think it's ok - not sure. Easy enough to hash it though, so maybe I'll do that?
Re. auth_url - I'm considering the weird edge case where the username/password changes - I think I'll go the hashing option.

Reply:
@conbrad hashing password - probably overkill, could probably just not use username and password in key.... but just in case.